packages/rum/src/domain/lifeCycle.ts

```diff
@@ -1,13 +1,14 @@
 import { Context, ErrorMessage } from '@datadog/browser-core'
 import { RumPerformanceEntry } from '../browser/performanceCollection'
-import { RawRumEvent, RumEvent } from '../types'
+import { ManuallyAddedError, RawRumEvent, RumEvent } from '../types'
 import { RawRumEventV2, RumEventV2 } from '../typesV2'
 import { RequestCompleteEvent, RequestStartEvent } from './rumEventsCollection/requestCollection'
 import { AutoActionCreatedEvent, AutoUserAction, CustomUserAction } from './rumEventsCollection/userActionCollection'
 import { View, ViewCreatedEvent } from './rumEventsCollection/viewCollection'
 
 export enum LifeCycleEventType {
   ERROR_COLLECTED,
+  MANUAL_ERROR_COLLECTED,
   PERFORMANCE_ENTRY_COLLECTED,
   CUSTOM_ACTION_COLLECTED,
```

Comment:
Why not `CUSTOM_ERROR_COLLECTED`? it would be more similar to action

Reply:
I think it could be confused with `ErrorSource.CUSTOM` (manual errors can have various sources)...